test/wsdl/default_namespace.wsdl

```diff
@@ -24,12 +24,6 @@
       <wsdl:output message="OutputMessage">
     </wsdl:output>
     </wsdl:operation>
-    <wsdl:operation name="prefixed-MyOperation">
```

Comment:
why is this being removed?

Reply:
In an earlier change I added unittests using a changed version of the existing `default_namespace.wsdl` with added operations that contained non identifier characters. I decided to move these to a new wsdl (`non_identifier_chars_in_operation.wsdl`) as to not possibly changing the behaviour of other tests (altough they didn't fail).